### ISSUE TITLE
Drupal 7 files

### ISSUE THREAD

**Issue by finnlewis** [IIED-org/pubs#123]
@cbrody @ncktrnr 

While migrating files and images, @cbrody and I were discussing the fact that there are a lot of files in the root of /sites/default/files/ which is certainly bad for our sanity, but may also create other problems.

This is worth noting: https://support.acquia.com/hc/en-us/articles/360005434833-Optimizing-file-paths-Organizing-files-in-subfolders

I stumbled upon it while reviewing: https://docs.acquia.com/cloud-platform/onboarding/preparing-launch/

Basically it sounds like Acquia recommend an upper limit of 2500 files. 

```
$ ls -l sites/default/files/| grep -v ^l | wc -l 
11895
```

Looks like we have a fair few more than that. 

Maybe we should try improving the file structure on the Drupal 7 site with the script that Acquia recommend: 

https://cskb.acquia.com/sites/default/files/2019-08/d7_migration_filepath.drush_.inc_.txt

**Comment by cbrody** [comment]
Thanks Finn, this script looks promising but it doesn't seem to take into account files that might be directly linked from text (e.g. body) fields - I think we could end up with a lot of broken links.

**Comment by finnlewis** [comment]
We have a problem with endoding of files. Some filenames have French characters in them. For some reason the migration appears to convert the accent e to a non-accent e, then it cannot find the file. Need to fix this!

**Comment by cbrody** [comment]
The migration process threw up a number of file missing errors. I've added a [Missing files tab](https://docs.google.com/spreadsheets/d/1LidUp0ZkfAi_RYqX1v17neoz2Xu8H0XxGPPZzgplPCY/edit#gid=309236424) to the inventory spreadsheet and revised the iied.org/admin/files view to allow filtering by filename and deletion – @ncktrnr do you think OK to go ahead and delete these from the file managed system?

**Comment by finnlewis** [comment]
Hi @cbrody 

The file migrations with French characters in the file migrations actually seem to work fine on my local, Ubuntu host with Lando. 

```
lando drush mim iied_d7_files --idlist=20881
```

![image](https://user-images.githubusercontent.com/326588/146930299-e422a5d9-b65d-4775-89a4-0db9916a128e.png)

**Comment by cbrody** [comment]
See related issue on iiedorg: https://github.com/IIED-org/iiedorg/issues/112

**Comment by finnlewis** [comment]
Hi @cbrody 

I think this one should have been addressed with https://github.com/IIED-org/pubs/pull/164

**Comment by cbrody** [comment]
Odd but there are currently over 12k files in iiedorg/prod/sites/default/files. Not sure how this has happened as there are just 2.5k files in the equivalent stage site directory. Don't think we have time / resource to re-open this issue, so closing for now as the other issues appear to have been resolved.
